Sources/Stream-Feed-UIKit-IOS/GetStreamActivityFeed/User Story/Activity Feed/Edit/EditPostViewController.swift

```diff
@@ -92,40 +92,66 @@ public final class EditPostViewController: UIViewController, BundledStoryboardLo
                 guard let self = self else { return }
                 switch compressionResult {
                 case .onSuccess(_):
-                    self.dismissAlertView()
+                    break
                 case .onStart:
                     guard let uploadedVideosItems = presenter?.mediaItems.filter({ $0.mediaType == .video }), uploadedVideosItems.count > 0 else { return }
                     self.compressedVideosCount += 1
                     self.showAlertWithProgressView()
                 case .onCancelled, .onFailure(_):
                     self.dismiss(animated: true)
+                
                 }
             }
             .store(in: &bag)
+        
+        presenter?.filesProgressSubject
+            .dropFirst()
```

Comment:
what does this dropFirst() do here ??

Reply:
can you please, add comment to explain ?

Reply:
@Jimmy-Camelan  `dropFirst()` method in **Combine** its equivalence to `skip(1)` in **RxSwift**.
both help us ignore the initial event when we start listening to data, useful when that initial event isn't important and we need to ignore it.

->We don't need to display the progress alert view when initializing the listener here so we skip the first event.